lms/djangoapps/onboarding/models.py

```diff
@@ -901,3 +901,51 @@ class RegistrationType(models.Model):
         related_name='registration_type'
     )
     choice = models.SmallIntegerField(default=1, null=False)
+
+
+class Education(models.Model):
+    """
+    Model to store user education information
+    """
+    linkedin_id = models.IntegerField(null=True, blank=True, unique=True)
+    user = models.ForeignKey(User, related_name='education',
+                             on_delete=models.CASCADE)
+    school_name = models.CharField(max_length=100, null=True, blank=True)
+    degree_name = models.CharField(max_length=100, null=True, blank=True)
+    start_month_year = models.DateField(null=True, blank=True)
+    end_month_year = models.DateField(null=True, blank=True)
+    description = models.CharField(max_length=500, blank=True, null=True)
+
+    def __unicode__(self):
+        return '%s %s' % (self.school_name, self.degree_name)
```

Comment:
Please use .format

Reply:
it was used like that in django documentation.
https://docs.djangoproject.com/en/1.8/ref/models/instances/#unicode

Reply:
But in PhilU we prefer to use format for this purpose,